boards/adafruit-metro-m4-express/include/periph_conf.h

```diff
@@ -0,0 +1,265 @@
+/*
+ * Copyright (C) 2024 ML!PA Consulting GmbH
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @ingroup     boards_adafruit-metro-m4-express
+ * @{
+ *
+ * @file
+ * @brief       Configuration of CPU peripherals for the Adafruit Metro M4 Express
+ *
+ * @author      Marian Buschsieweke <marian.buschsieweke@posteo.net>
+ */
+
+#ifndef PERIPH_CONF_H
+#define PERIPH_CONF_H
+
+#include "periph_cpu.h"
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+/**
+ * @name    Core clock frequency
+ * @{
+ */
+#ifndef CLOCK_CORECLOCK
+#define CLOCK_CORECLOCK     MHZ(120)
+#endif
+/** @} */
+
+/**
+ * @name    32kHz Oscillator configuration
+ * @{
+ */
+#define EXTERNAL_OSC32_SOURCE                   1
+#define INTERNAL_OSC32_SOURCE                   0
+#define ULTRA_LOW_POWER_INTERNAL_OSC_SOURCE     0
+/** @} */
+
+/**
+ * @brief Enable the internal DC/DC converter
+ *        The board is equipped with L_EXT (L1) of 10uH at VSW pin.
+ */
+#define USE_VREG_BUCK       (1)
+
+/**
+ * @name ADC Configuration
+ * @{
+ */
+#define ADC_PRESCALER                       ADC_CTRLA_PRESCALER_DIV128
```

Comment:
Better use

```suggestion
#define ADC_GCLK_SRC                        SAM0_GCLK_PERIPH
#define ADC_PRESCALER                       ADC_CTRLA_PRESCALER_DIV8
```
Using 120 MHz main clock (default) is out of spec for the ADC peripheral, the input GCLK (before prescaler!) must be below 100 MHz.

This has been wrong for the other boards too, I fixed this in #20622 but should probably better make a seprate PR for that.